Add New Relic monitor scripts

jmhooper opened the pull request:
**Why**: So that we can use New Relic browser scripting to monitor our
app.

zachmargolis commented on package.json:
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "identity-monitor",
+  "version": "1.0.0",
+  "description": "Automated tests for Login.gov",
```
should we add `"private": true` to prevent accidentally publishing to npm?

zachmargolis commented on new_relic_scripts/totp_sign_in.js.erb:
```diff
@@ -0,0 +1,151 @@
+}
+
+// Ref: https://github.com/guyht/notp/blob/master/index.js
+function generateTOTP() {
```
since we're already pre-processing this file, would it make sense to inline the source of the `notp` gem? ex something like:

```erb
(function(module, exports, require) {
  <%= File.read('node_modules/notp/index.json') %>
})(module, exports, require)
```

jmhooper replied:
Thought about that. I decided to do it this way for now with plans to use something like webpack to bundle it later

zachmargolis commented on bin/build_nr_scripts.sh:
```diff
@@ -0,0 +1,17 @@
+source .env
```
I don't think this worked for me? Or at least it wasn't visible to the ruby process? My JS file ended up with a bunch of blank secrets after I updated the `.env`

I tried `<%= require 'dotenv/load' %>` to the top of the ERB and that made it work for me

zachmargolis commented on Makefile:
```diff
@@ -20,4 +20,7 @@ lint: $(CONFIG)
 test:
 	bundle exec rspec
 
+build_nr_scripts:
```
since this is a pretty direct 1:1 translation, what if we used makefiles for dependency stuff?

```
new_relic_scripts_out/totp_sign_in.js: new_relic_scripts/totp_sign_in.js.erb .env
  ./bin/build_nr_scripts.sh $< > $@
```

zachmargolis replied:
I could have the Makefile magic variables wrong, I just thought they'd be nice, but the gist is the same

jmhooper commented:
@zachmargolis: I went ahead and added the tooling to build out the scripts with webpack. Copying the bundles into the New Relic UI feels weird, but it helps us keep our source cleaner. We can also configure Circle or something to build and deploy them to New Relic for use when we merge changes if we want? 

I also added ESLint to keep things tidy.
